src/sentry/integrations/slack/unfurl/discover.py

```diff
@@ -105,7 +107,13 @@ def unfurl_discover(
     links: list[UnfurlableUrl],
     user: User | None,
 ) -> UnfurledUrl:
-    orgs_by_slug = {org.slug: org for org in integration.organizations.all()}
+    org_integrations = integration_service.get_organization_integrations(
+        integration_id=integration.id
+    )
+    organizations = Organization.objects.filter(
```

Comment:
@leeandher should this be using `organization_service`?

Reply:
IFF it runs in control or in both control and region.